src/invidious/routes/misc.cr

```diff
@@ -5,24 +5,22 @@ class Invidious::Routes::Misc < Invidious::Routes::BaseRoute
     user = env.get? "user"
 
     case preferences.default_home
-    when "Popular"
-      env.redirect "/feed/popular"
-    when "Trending"
-      env.redirect "/feed/trending"
-    when "Subscriptions"
+    when Settings::HomePages::Popular ; env.redirect "/feed/popular"
+    when Settings::HomePages::Trending; env.redirect "/feed/trending"
+    when Settings::UserHomePages::Subscriptions
```

Comment:
Please use the `then` keyword instead of `;`

`when Settings::HomePages::Popular then env.redirect "/feed/popular"`

Reply:
Oh, didn't know about that, thanks! Was it recently added?